js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -1213,7 +1213,11 @@ var TreeViewBase = HierarchicalCollectionWidget.inherit({
     _animateNodeContainer: function(node, state, e) {
         var $node = this._getNodeElement(node),
             $nodeContainer = $node.children("." + NODE_CONTAINER_CLASS),
-            nodeHeight = $nodeContainer.height();
+            nodeHeight;
+
+        // TODO: The height of node container is should be used when the container is shown (T606878)
```

Comment:
TODO -> NOTE?

Reply:
Fixed in https://github.com/DevExpress/DevExtreme/commit/aaec43849bf103386fc674325f806020b50e64d1